packages/backend/src/modules/tvl2/utils/getValuesConfigHash.ts

```diff
@@ -0,0 +1,30 @@
+import { createHash } from 'crypto'
+import { assert } from '@l2beat/backend-tools'
+import {
+  AmountConfigEntry,
+  CoingeckoPriceConfigEntry,
+} from '@l2beat/shared-pure'
+import { createAmountId } from './createAmountId'
+import { createValueId } from './createValueId'
+
+export function getValuesConfigHash(
+  amountConfigs: AmountConfigEntry[],
+  priceConfigs: CoingeckoPriceConfigEntry[],
+): string {
+  const input: string[] = []
```

Comment:
I'd make it more explicit
```suggestion
  const valueIds: string[] = []
```